app/helpers/posts_helper.rb

```diff
@@ -0,0 +1,6 @@
+module PostsHelper
+  def post_liked?(post, user)
+    @like = Like.where(post_id: post.id, user_id: user.id)
+    @like.present?
```

Comment:
No need to assign it to any variable, You can directly use `Like.where(post_id: post.id, user_id: user.id).present?` with a single line. Check once.